commands/ssh-key/get/get.go

```diff
@@ -0,0 +1,108 @@
+package get
+
+import (
+	"errors"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/profclems/glab/internal/glrepo"
+	"github.com/profclems/glab/pkg/iostreams"
+	"github.com/profclems/glab/pkg/prompt"
+	"github.com/profclems/glab/pkg/utils"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+type GetOpts struct {
+	HTTPClient func() (*gitlab.Client, error)
+	IO         *iostreams.IOStreams
+	BaseRepo   func() (glrepo.Interface, error)
+
+	KeyID int
+}
+
+func NewCmdGet(f *cmdutils.Factory, runE func(*GetOpts) error) *cobra.Command {
+	opts := &GetOpts{
+		IO: f.IO,
+	}
+	cmd := &cobra.Command{
+		Use:   "get <key-id>",
+		Short: "Gets a single key",
+		Long:  "Returns a single SSH key specified by the ID",
+		Example: heredoc.Doc(`
+		# Get ssh key with ID as argument
+		$ glab ssh-key get 7750633
+
+		# Interactive
+		$ glab ssh-key get
+		`),
+		Args: cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.HTTPClient = f.HttpClient
+			opts.BaseRepo = f.BaseRepo
+
+			if len(args) == 0 && !opts.IO.IsOutputTTY() {
+				return cmdutils.FlagError{Err: errors.New("<key-id> argument is required running in non-ttys")}
+			}
+
+			if len(args) == 1 {
+				opts.KeyID = utils.StringToInt(args[0])
+			}
+
+			if runE != nil {
+				return runE(opts)
+			}
+
+			return addRun(opts)
+		},
+	}
+
+	return cmd
+}
+
+func addRun(opts *GetOpts) error {
```

Comment:
You probably meant `getRun` over here

Reply:
Ahh, thanks! I copied from the `add` and forgot to fix the function name. 

I was actually dozing off lol